token/program-2022/src/instruction.rs

```diff
@@ -469,6 +469,32 @@ pub enum TokenInstruction {
     ///   None
     ///
     InitializeImmutableOwner,
+    /// Convert an Amount of tokens to a UiAmount `string`, using the given mint.
+    ///
+    /// Fails on an invalid mint.
+    ///
+    /// Return data can be fetched using `sol_get_return_data` and deserialized with
+    /// `String::from_utf8`.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   0. `[]` The mint to calculate for
```

Comment:
Yeah, I can see a use for that. I wonder if that should/could be part of the iToken extension? Or would it need to be supported by both token programs?
I'll merge this as-is for now, but we can update this interface while building out the extension, if it seems like it would be more ergonomic